docs/learning/ui/UIGlossary.md

```diff
@@ -6,20 +6,16 @@ tableRowAnchors: true
 
 |Term | Definition
 |------------|------------|
-|**Backstage**|A menu used to open Frontstages and launch commands. It is opened by clicking or pressing the App button. It can open a full-screen overlay presenting application settings and data management to the user.
-|**Content&nbsp;Control**|A class that specifies the React component to display for a Content View.
-|**Content&nbsp;Group**|A collection of Content Controls.
-|**Content&nbsp;Layout**|A layout configuration of Content Views.
-|**Content&nbsp;Manipulation&nbsp;Tools**|These tools are located in the left side of the content area. These tools are used to Select, Create, Update, and Delete data.
-|**Content&nbsp;View**|A representation of an iModel's data.
-|**Dialog**|A small temporary window of options presented to the user.
-|**Frontstage**|A full-screen configuration designed to enable the user to accomplish a task. There are three types: Primary, Nested and Modal.
-|**Keyboard&nbsp;Shortcut**|A keystroke or combination of keystrokes used to launch a command or tool.
-|**Navigation&nbsp;Aid**|A user interface control that moves the user's perspective around within a specific Content View.
-|**Panel&nbsp;Section**|An area of within a Stage Panel that can contain a collection of Widgets. There are two designated sections in each panels: Start and End.
-|**Stage&nbsp;Panel**|A panel defined as part of a Frontstage that displays to the left, right, above or below the content area.
-|**Status&nbsp;Bar**|The Footer area of the application window that gives the user feedback about the state of an application.
-|**Status&nbsp;Field**|An area of the Status Bar assigned to display specific feedback about the active application. Also referred to as **Status&nbsp;Item**
-|**Tool&nbsp;Settings**|Properties that modify the behavior and outcome of the active tool. Typically docked as a bar at the top of the application window.
-|**View&nbsp;Navigation&nbsp;Tools**|These tools are located in the right side of the content area. These tools are used to navigate and modify the view parameter such as rotation and zoom level.
-|**Widget**|A collection of UI components tied to a particular section that allows the user to view and/or modify data relevant to their current context.
+|**Backstage**|The main navigation menu of the application used to switch between frontstages, open overlays, or launch custom commands. You can use [BackstageComposer]($appui-react) to set up a backstage that displays an overlay menu along the left edge of the application.|
+|**Content&nbsp;Control**|Control that is displayed in a content layout. You can use [ContentControl]($appui-react) to specify a custom React component.|
+|**Content&nbsp;Group**|Defines which content controls are rendered and how in the content area of a frontstage. You can use [ContentGroup]($appui-react) to describe a collection of content controls and their content layout.|
```

Comment:
"Defines which content controls are rendered and how in the content area of a frontstage" is a bit awkward. 

Is the "which" necessary?
> Defines how content controls are rendered in the content area of a frontstage.

Reply:
Eh, circling back I can live with this phrasing. Unless you like my suggestion better.

Reply:
Updated.